File: Benchmarks/normal/python/flask/testcode/benchmark_test_09735.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest09735():
    field_value = request.form.get('field', '')
    data = f'{field_value:.200s}'
    return jsonify({'status': 'ok'}), 200, {'Access-Control-Allow-Origin': str(data)}

File: Benchmarks/normal/java/jakarta/testcode/BenchmarkTest09735.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import jakarta.ws.rs.*;
import jakarta.ws.rs.core.Context;
import jakarta.ws.rs.core.MediaType;
import jakarta.ws.rs.core.Response;

@Path("/")
public class BenchmarkTest09735 {

    @GET
    @Path("/BenchmarkTest09735")
    @Produces(MediaType.APPLICATION_JSON)
    public Response BenchmarkTest09735(@HeaderParam("Origin") String origin, @Context HttpServletRequest request, @Context HttpServletResponse response) throws Exception {
        String originValue = origin != null ? origin : "";
        String data = String.join(" ", originValue.split("\\s+"));
        byte[] buf = new byte[Integer.parseInt(data)];
        return Response.ok("{\"ready\":true}", MediaType.APPLICATION_JSON).build();
    }
}
